clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -620,6 +620,11 @@ void CudaToolChain::addClangTargetOptions(
       CC1Args.push_back("-fgpu-rdc");
   }
 
+  if (DeviceOffloadingKind == Action::OFK_SYCL) {
+    toolchains::SYCLToolChain::AddSYCLIncludeArgs(getDriver(), DriverArgs,
+                                                  CC1Args);
+  }
+
```

Comment:
Seems out of place here - would be better in `AddCudaIncludeArgs`?

Reply:
I started from that but AddCudaIncludeArgs is used only for host code compilation.